INSVCTwoFluidStaggeredHierarchyIntegrator.cpp

```diff
@@ -418,6 +417,22 @@ INSVCTwoFluidStaggeredHierarchyIntegrator::integrateHierarchy(const double curre
     const double dt = new_time - current_time;
     const double C = d_rho / dt;
 
+    // Need to fill in ghost cells for theta to do interpolation
+    {
+        using ITC = HierarchyGhostCellInterpolation::InterpolationTransactionComponent;
+        std::vector<ITC> ghost_cell_comp(1);
+        ghost_cell_comp[0] = ITC(thn_cc_idx,
+                                 "CONSERVATIVE_LINEAR_REFINE",
+                                 false,
+                                 "NONE",
+                                 "LINEAR",
+                                 true,
+                                 nullptr); // defaults to fill corner
+        HierarchyGhostCellInterpolation ghost_cell_fill;
+        ghost_cell_fill.initializeOperatorState(ghost_cell_comp, d_hierarchy, 0, d_hierarchy->getFinestLevelNumber());
+        ghost_cell_fill.fillData(0.0);
+    }
+
```

Comment:
This was the reason the solver wasn't iterating. We were filling in junk when building the RHS.

Reply:
Great, this explains why it was producing NaNs previously.